config.tutorial.yaml

```diff
@@ -13,89 +13,77 @@ scenario:
   simpl: ['']
   ll: ['copt']
   clusters: [6]
-  opts: [Co2L-24H]
+  opts: [Co2L-3H]
 
-countries: ["NG", "BJ"]
+countries: ["africa"]
```

Comment:
@giacfalk, It seems you use the default `config.yaml` and renamed it to `config.tutorial.yaml`. This can cause issues as the tutorial was designed to download a minimal example that required some particular naming. I would recommend:
1. delete the current `config.tutorial.yaml`
2. copy the original `config.tutorial.yaml`
3. only add the parts you added regarding the load options